mapentity/tests/test_functional.py

```diff
@@ -235,25 +235,68 @@ def test_api_list_for_model(self):
             return  # Abstract test should not run
         self.login()
 
-        obj = self.modelfactory()
+        obj = self.modelfactory.create(geom='POINT(0 0)', name='name')
         list_url = '/api/{modelname}s/'.format(modelname=self.model._meta.module_name)
         response = self.client.get(list_url)
         self.assertEqual(response.status_code, 200)
-        first_result = json.loads(response.content)[0]
+        result = json.loads(response.content)
+        self.assertEqual(len(result), 1)
+        first_result = result[0]
+        self.assertItemsEqual(first_result.keys(), ('id', 'name', 'geom', 'date_update'))
+        self.assertEqual(first_result['id'], obj.pk)
+        self.assertEqual(first_result['name'], 'name')
+        self.assertEqual(first_result['geom'], 'POINT (0.0000000000000000 0.0000000000000000)')
+
+    def test_api_geojson_list_for_model(self):
+        if self.model is None:
+            return  # Abstract test should not run
+        self.login()
+
+        obj = self.modelfactory.create(geom='POINT(0 0)', name='name')
+        list_url = '/api/{modelname}s/.geojson'.format(modelname=self.model._meta.module_name)
```

Comment:
I didn't know about this notation !

This would be equivalent to putting Accept request header I guess.